tests/pyccel/test_pyccel.py

```diff
@@ -730,6 +757,15 @@ def test_multiple_results(language):
 def test_elemental(language):
     pyccel_test("scripts/decorators_elemental.py", language = language)
 
+#------------------------------------------------------------------------------
+@pytest.mark.cuda
+@pytest.mark.gpu
```

Comment:
on my machine chaining these markers only works if you specify the second one.
on another note I think you should only use `gpu` for these tests.

Reply:
In case I use the gpu marker, my tests are not going to run , so the bot is going to tell me I didn't test some part of my code

Reply:
But I think that's not a big 
problem

Reply:
> on my machine chaining these markers only works if you specify the second one.

Is that not what is intended? On the CI we want to run all things tagged `cuda` but not `gpu`.
The default is changed here: https://github.com/pyccel/pyccel-cuda/pull/42/files#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R65-R71
So you can't accidentally run GPU tests